code/game/objects/structures/morgue.dm

```diff
@@ -132,7 +132,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 	playsound(src, 'sound/effects/roll.ogg', 5, 1)
 	playsound(src.loc, 'sound/items/deconstruct.ogg', 50, 1)
 	for(var/atom/movable/AM in connected.loc)
-		if(!AM.anchored || AM == connected || isliving(AM))
+		if(!AM.anchored || AM == connected && (istype(AM, /mob) && !isliving(AM)))

```

Comment:
connected is a tray.

so if it's not anchored (which ghosts aren't), or if it's a tray, and the tray is a mob, and the tray isn't living, move inside.

Reply:
```
if(ismob(AM) && !isliving(AM))
		continue
if(!AM.anchored || AM == connected)
	AM.forceMove(src)
```